subsys/bluetooth/controller/ll_sw/nordic/lll/lll.c

```diff
@@ -454,15 +540,15 @@ static int prepare(lll_is_abort_cb_t is_abort_cb, lll_abort_cb_t abort_cb,
 
 		if (next) {
 			/* check if resume requested by curr */
-			ret = event.curr.is_abort_cb(NULL, 0, event.curr.param,
+			err = event.curr.is_abort_cb(NULL, 0, event.curr.param,
 						     &resume_cb, &resume_prio);
-			LL_ASSERT(ret);
+			LL_ASSERT(err);
```

Comment:
not really related to this PR, but I think this ASSERT should be removed; depending on how the ASSSERT is implemented this may stop the flow here.